cmd/dist/common.go

```diff
@@ -12,10 +17,31 @@ import (
 	"github.com/containerd/containerd/remotes"
 	"github.com/containerd/containerd/remotes/docker"
 	imagesservice "github.com/containerd/containerd/services/images"
+	"github.com/crosbymichael/console"
+	"github.com/pkg/errors"
 	"github.com/urfave/cli"
 	"google.golang.org/grpc"
 )
 
+var registryFlags = []cli.Flag{
+	cli.BoolFlag{
+		Name:  "insecure,k",
+		Usage: "Skip SSL certificate validation",
+	},
+	cli.BoolFlag{
+		Name:  "plain-http",
+		Usage: "Allow connections using plain HTTP",
+	},
+	cli.StringFlag{
+		Name:  "user",
```

Comment:
Was basing it off the flags used for `curl`. Normally we want to discourage having the password there but it is just providing a way to use it. Also have the refresh token option which is separate from the basic auth credentials, so using a generic term is not ideal.